### DISCUSSION TITLE
Retrieve file path

### DISCUSSION THREAD

**Discussion by tweetysat** [Jaaneek/useFilePicker#37] [Q&A] [answered]
I'm using your hook with 

```
const [openFileSelector, { loading, plainFiles }] = useFilePicker({
    accept: '.exe',
    multiple: false,
    readFilesContent: false
  });
```

Is is working fine, I can get the file name with the plainFiles object.

But is there a way to also retrieve the path of the file ?

**Comment by MrKampla** [comment, marked as answer]
I don't think so, browsers hide information like this for privacy & security reasons
